<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>640</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>440</width>
    <height>640</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>440</width>
    <height>640</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/music_player/music-2935457_640.png</normaloff>:/music_player/music-2935457_640.png</iconset>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#Widget{
	background-image: url(:/music_player/OIP-C.png);
}

*{
background-color:transparent;
}
QPushButton:hover{
rgb(176, 206, 255);
}
/*QSlider::groove:horizontal{ 
		height: 12px; 
		left: 0px; 
		right: 0px; 
		border:0px;    //指定无边框
	    border-radius:6px;    //指定圆角
		background:rgba(0,0,0,50);
 } 
 //滑块的美化
 QSlider#playCourseSlider::handle:horizontal{ 
		width:  50px; 
		height: 50px; 
		margin-top: -20px; 
		margin-left: 0px; 
		margin-bottom: -20px; 
		margin-right: 0px; 
 		border-image:url(:/res/images/setting_slider_handle.png);
} 
//已滑过的进度美化
QSlider::sub-page:horizontal{
	   background:rgba(80,166,234,1);
}*/</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>550</y>
     <width>401</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
border:none;
}
QPushButton:hover
{
	background-color:rgb(171, 233, 255);
}</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QPushButton" name="pushButton_file_name">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>42</width>
        <height>42</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>42</width>
        <height>42</height>
       </size>
      </property>
      <property name="toolTip">
       <string/>
      </property>
      <property name="styleSheet">
       <string notr="true">/*background-image: url(:/asset/playfile.png);
background-image: url(:/music_player/shutdown-2935465_640.png);
background-repeat: no-repeat;
background-size: cover; 
*/</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_file_last">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>42</width>
        <height>42</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>42</width>
        <height>42</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">/*background-image: url(:/asset/playfile.png);
background-repeat: no-repeat;
background-size: cover; 
*/</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_file_play">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>42</width>
        <height>42</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>42</width>
        <height>42</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">/*background-image: url(:/images/620.bmp);
background-repeat: no-repeat;  
*/
*{border:none;}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_file_next">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>42</width>
        <height>42</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>42</width>
        <height>42</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">/*background-image: url(:/asset/next.png);*/</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_file_volume">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>42</width>
        <height>42</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>42</width>
        <height>42</height>
       </size>
      </property>
      <property name="font">
       <font>
        <bold>false</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="MyListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>421</width>
     <height>421</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文楷体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QListWidget#listWidget{
&quot;QListWidget { font-size: 18pt; font-family: 华文新魏;
background-color:transparent;
}ui-&gt;listWidget-&gt;setStyleSheet(&quot;QListWidget { font-size: 18pt; font-family: 华文新魏; }&quot;);
ui-&gt;listWidget-&gt;setStyleSheet(&quot;QListWidget { border: 10px solid black; }&quot;);</string>
   </property>
  </widget>
  <widget class="QSlider" name="playCourseSlider">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>500</y>
     <width>311</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;进度条&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QSlider#playCourseSlider::handle{  
 width: 100px !important;  
}</string>
   </property>
   <property name="pageStep">
    <number>5</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="volume_slider">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>480</y>
     <width>22</width>
     <height>61</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;音量&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>450</y>
     <width>391</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="current_label">
      <property name="text">
       <string>00:00</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="total_label">
      <property name="text">
       <string>00:00</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyListWidget</class>
   <extends>QListWidget</extends>
   <header>mylistwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
